src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
```

Comment:
Redundant empty space

Reply:
I think this part of code is on origin branch, i dont have it in my IntelliJ IDEA.